src/base.c

```diff
@@ -377,10 +377,11 @@ int cw_authenticate(const char *service, const char *username, const char *passw
 	};
 
 	rc = pam_start(service, creds.username, &convo, &pam);
-	if (rc == PAM_SUCCESS)
+	if (rc == PAM_SUCCESS) {
 		rc = pam_authenticate(pam, PAM_DISALLOW_NULL_AUTHTOK);
 		if (rc == PAM_SUCCESS)
 			rc = pam_acct_mgmt(pam, PAM_DISALLOW_NULL_AUTHTOK);
+	}
```

Comment:
Are we 100% sure about this?  I get that the whitespace implies that the braces should be there, but it _does_ change the semantics to add them.